software/gamelogic/gamelogic.cpp

```diff
@@ -57,9 +57,11 @@ int main () {
             }
         }
         else { // dealing with UI / BT
+            cout << "i'm here" << endl;
             Init_RS232();
             Init_RS232_WIFI();
             blankscreen( BLUE );
+            cout << "past init" << endl;
             int player_num;

```

Comment:
I prefer num_players, player_num is a tad confusing

Reply:
done